Taskfile.yml

```diff
@@ -0,0 +1,163 @@
+# https://taskfile.dev/
+
+version: '3'
+
+vars:
+  BINARY_PATH: "${HOME}/.terraform.d/plugins/registry.terraform.io/opslevel/opslevel"
```

Comment:
This is not the same as the original location - its very critical that we get this correct otherwise it doesn't use the newly built binary.
```${PROVIDER_PATH}/${HOSTNAME}/${NAMESPACE}/${NAME}/${VERSION:1}/${OS_ARCH}/${BINARY}"```

Reply:
Yes -  good call! Fix in next commit